time_difference.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |gem|
   gem.version       = "0.7.0"
   gem.license = 'MIT'
 
-  gem.add_runtime_dependency('activesupport', '~> 5.1')
```

Comment:
This change looks unrelated.

Reply:
yea.. but there was no need to hard code the activesupport version here, since this wont support rails 6 now.

Reply:
Maybe, but it's better to make unrelated changes in separate PRs.

Reply:
ok.. i'll create seaprate PR then

Reply:
https://github.com/tmlee/time_difference/pull/51

Reply:
https://github.com/tmlee/time_difference/pull/52

Reply:
Nice! I think that we can close this one in favor of these two.

Reply:
Or, honestly, you could change this one to something main, like precision, without unrelated changes, like `activesupport` version, and create only one separate for your case, with this `activesupport` version change.